exercises/K-functions-parameters/exercise4.js

```diff
@@ -1,5 +1,8 @@
-// Declare your function first
 
-// Call the function and assign to a variable `sum`
+function add(num1, num2){
+     return sum = num1 + num2;
```

Comment:
This may work but it is actually incorrect. Can you figure out what is wrong with this?